rx-java/src/main/asciidoc/vertx-rx/java/sql.adoc

```diff
@@ -0,0 +1,36 @@
+== Transaction management with RxJava
+
+Managing transactions with the Rxified API `SQLConnection` requires quite a lot of boilerplate code.
+
+Vert.x provides observable __transformers__ that you can apply to your flows with `compose` in order to make them transactional:
```

Comment:
I'd put "observable transformers" in the same emphasis.

Reply:
You may also link to the `compose` operator in RxJava for the interested reader.